core/uimarket/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
In my opinion when we need a code, file or.. we should add it. so this can be deleted